app/modules/transcribe/modals/variants.controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+require('./modals.module.js')
+    .controller('VariantsController', VariantsController);
+
+// @ngInject
+function VariantsController($modalInstance, AnnotationsFactory) {
+    var vm = this;
+    vm.close = close;
+    vm.data = AnnotationsFactory.list();
+    vm.variants = getVariants;
+
+    function close(result) {
+        $modalInstance.close(result);
+    }
+
+    function getVariants() {
```

Comment:
Yes I forgot in there when I was trying to display the actual possible variants in the html.